Sources/PaymentsCore/Networking/URLCacheTestable.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+protocol URLCacheTestable {
```

Comment:
Since we are using the protocol in the real implementation and not just testing do we think it's worth removing the test portion of the name? Or did we mean test in a different way?

Reply:
So I thought that for protocols that _only_ exist for the purpose of injecting mocks, we could name them `Testable`? We currently have `URLSessionProtocol` which exists for the same purpose, so that we can inject a mock to replace `URLSession.shared`. But the name `URLSessionProtocol` doesn't follow the "adjective ending in -able or -ing" Swift language style guide.

I thought `Testable` might make it more clear what function the protocol serves. But it was more-so a trial to see what y'all think. 

Maybe `Mockable`? Or do you prefer just calling it `Protocol`?

Reply:
I like that idea! Updated